.github/workflows/ci-lint.yml

```diff
@@ -18,11 +18,23 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Run install script for all targets
-        run: make install
-
-      - name: Run clippy check for all targets
-        run: make clippy
+      - name: Run clippy check host
+        run: cargo clippy -p raiko-host -p raiko-lib -p raiko-core -p harness -p raiko-tasks --all-features  -- -D warnings
+      - name: Run clippy check risc0
+        run: |
+          cargo clippy -p risc0-builder --all-features -- -D warnings
+          cargo clippy -p risc0-driver --features enable -- -D warnings
+          cd ./provers/risc0/guest
+          cargo clippy --all-features -- -D warnings
+      - name: Run clippy check sp1
+        run: |
+          cargo clippy -p sp1-builder --all-features -- -D warnings
+          cargo clippy -p sp1-driver --features enable -- -D warnings
+          cd ./provers/sp1/guest
+          cargo clippy --all-features -- -D warnings
+      - name: Run clippy check sgx
+        run: |
+          cargo clippy -p sgx-guest -p sgx-prover -p raiko-setup --all-features -- -D warnings
```

Comment:
There needs to be an install step because clippy will still compile the projects so they can't actually compile without the tooling present